backend/src/main/java/corea/feedback/repository/DevelopFeedbackRepository.java

```diff
@@ -0,0 +1,23 @@
+package corea.feedback.repository;
+
+import corea.feedback.domain.FeedbackKeyword;
+import corea.feedback.domain.DevelopFeedback;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface DevelopFeedbackRepository extends JpaRepository<DevelopFeedback, Long> {
+
+    @Query("SELECT r.keywords FROM DevelopFeedback r WHERE r.receiver.id = :receivedId")
+    List<List<FeedbackKeyword>> findAllKeywordsByRevieweeId(long receivedId);
```

Comment:
```suggestion
    List<List<FeedbackKeyword>> findAllKeywordsByReceiverId(long receivedId);
```

reviewer 가 receiver가 되는 게 맞나요?
제가 약간의 무지성 suggestion을 드리는 것 같아서 희선씨도 크로스체크 후 수정 부탁드리겠습니다! 🙇🏻‍♀️

Reply:
맞습니다! 해당 부분은 Query 로 동작해서 놓쳤네용 감사합니당.